.github/workflows/refresh.yml

```diff
@@ -0,0 +1,20 @@
+# File: .github/workflows/refresh.yml
+name: Refresh Static Content
+
+on:
+  schedule:
+    - cron:  '0 8 * * *' # Runs every day at 8am
+
+jobs:
+  refresh:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger GitHub pages rebuild
+        run: |
+          curl --fail --request POST \
+            --url https://api.github.com/repos/${{ github.repository }}/pages/builds \
+            --header "Authorization: Bearer $GITHUB_TOKEN"
+        env:
+          # You must create a personal token with repo access as GitHub does
+          # not yet support server-to-server page builds.
+          USER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```
Since we're using the automatically provided GITHUB_TOKEN, we don't have to inject anything into the environment here. At least, that's the hope. We may have to test this a couple times. 😄